Initial attempt at scala automated comments for two-fer (#1148)
exercism/website-copy: automated-comments/scala/general/no_return_type.md

> A return type should be specified for all public members. Including the return type allows the compiler to validate that the expected return type. And, ensures that changing Scala type inference maintains binary compatibility. 

Reviewer: ```suggestion
A return type should be specified for all public members. Including the return type allows the compiler to validate the expected return type. It also ensures that changing Scala type inference maintains binary compatibility. 
```

Reviewer 2: ```suggestion
Specify a `return` type for all public members. Including the return type allows the compiler to validate the expected `return` type. It also ensures that changing Scala type inference maintains binary compatibility. 
```
